blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,291 @@
+[Goroutines](https://go.dev/tour/concurrency/1) to
+[async/await](https://en.wikipedia.org/wiki/Async/await), we have been exploring better ways to
+express non-linear programming. 
+
+But preflights and inflights are just the tip of the iceberg in terms of what a language for the
+cloud can offer. Think about API endpoints and clients, telemetry, metrics, alarms, data schemas. We
+have plans to bake many of these capabilities into the language and the standard library so that
+developers can truly focus on building their applications.
+
+### A standard library for the cloud
+
+The second piece of the puzzle is what we call the **Wing Cloud Library** (or WCL). This is a
+library of classes that represent common resources for building cloud applications. You can think of
+the WCL as a [standard library](https://en.wikipedia.org/wiki/Standard_library) for the cloud.
+Similarly to how standard libraries in traditional languages abstract the operating system, WCL
+abstracts cloud providers and services.
+
+We believe the cloud has finally matured to the point where it is possible to establish these
+standard abstractions and that they will carry over successfully across public and private cloud
+providers. Even with traditional standard libraries we didn't always have that level of abstraction.
+When I started programming, I needed to know exactly which operating system, and even which file
+system, I was using in order to write a file. Now, I can just call `writeFile()` and it will work
+across all platforms. Other projects in the industry such as
+[wasi-cloud-core](https://github.com/WebAssembly/wasi-cloud-core) have also reached a similar
+conclusion.
+
+The WCL is designed from the ground up to support multiple targets for each resource. Classes in the
+library only define an abstract API, and it is possible to inject concrete implementations at
+compile time. This powerful mechanism means that code written against these APIs is **portable
+across cloud providers and provisioning engines**.
+
+And of course, we think of WCL as the foundation for an ecosystem of cloud libraries. The
+programming model is open and extensible and anyone will be able to publish libraries and implement
+backends for existing resources based on the needs of their environment.
+
+The WCL is a major project and in its early days. You can see the coverage of resources and targets
+through this [compatibility
+matrix](https://www.winglang.io/docs/standard-library/compatibility-matrix) and make sure to "+1"
+the relevant GitHub issue if coverage is missing.
+
+![](./assets/matrix.png)
+
+Currently, WCL only supports Winglang, but we are interested in offering it in other languages as
+well. The library is built using CDK technologies such as [JSII](https://github.com/aws/jsii) and
+[constructs](https://github.com/aws/constructs), so technically it is possible to publish it to
+other languages.
```

Comment:
I would consider removing this CDK and JSII part, the implementation details here are not relevant here IMO
```suggestion
well. 
```